it/src/test/scala/com/wavesplatform/it/sync/transactions/IssueTransactionSuite.scala

```diff
@@ -78,9 +78,9 @@ class IssueTransactionSuite extends BaseTransactionSuite with TableDrivenPropert
   val invalidAssetValue =
     Table(
       ("assetVal", "decimals", "message"),
-      (0l, 2, "negative amount"),
+      (0l, 2, "non-positive amount"),
```

Comment:
this breaks backward compatibility.
not only we change message, but also error code, I don't think UI team willbe ok with this.